src/Nethermind/Nethermind.Merge.Plugin/BlobTransactionsDbCleaner.cs

```diff
@@ -55,6 +58,10 @@ private void CleanProcessedTransactionsDb(long newlyFinalizedBlockNumber)
 
             if (_logger.IsDebug) _logger.Debug($"Cleaned processed blob txs from block {_lastFinalizedBlock} to block {newlyFinalizedBlockNumber}");
 
+            _db.Compact();
```

Comment:
Can we guard the compaction not use to many resources? (Threads? I/O?) So it won't starve anything else in block processing.